tests/functional/test_splash.py

```diff
@@ -40,3 +42,55 @@ def test_splash_screen_running(pyi_builder_spec, capfd, monkeypatch, build_mode,
     out, err = capfd.readouterr()
     assert 'SPLASH: splash screen started' in err, \
         f"Cannot find log entry indicating start of splash screen in:\n{err}"
+
+
+# Check that splash-enabled program properly exits (e.g., there are no crashes in bootloader during shutdown).
+# This is not coveved by the `test_splash_screen_running`, because that one terminates the program.
+# There are three variants of this test:
+# - splash screen is kept running until the end
+# - user imports pyi_splash, which calls pyi_splash.close() via atexit()
+# - user imports pyi_splash and calls pyi_splash.close()
+@pytest.mark.skipif(is_darwin, reason="Splash screen is not supported on macOS.")
+@pytest.mark.xfail(is_musl, reason="musl + tkinter is known to cause mysterious segfaults.")
```

Comment:
Could we scope these to the whole file? i.e. Global variable `pytestmark = [pytest.mark.skipif(is_darwin, ...), pytest.xfail(is_musl, ...)]`

Reply:
Yeah, good idea.